app/services/users.js

```diff
@@ -0,0 +1,29 @@
+const bcrypt = require('bcrypt');
+
+const User = require('../models/user');
+const logger = require('../logger/index');
+const errors = require('../errors');
+
+exports.createUser = async payload => {
+  try {
+    const [user, created] = await User.create({ where: { email: payload.email }, defaults: payload });
```

Comment:
You are just creating the user, you should not use the where here, you can directly pass the `payload` object to it and simply receive the created user.